tests/release/pipelines/rh_advisories.go

```diff
@@ -8,23 +8,23 @@ import (
 	"regexp"
 	"time"
 
-	appservice "github.com/konflux-ci/application-api/api/v1alpha1"
-	corev1 "k8s.io/api/core/v1"
 	ecp "github.com/enterprise-contract/enterprise-contract-controller/api/v1alpha1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	appservice "github.com/konflux-ci/application-api/api/v1alpha1"
 	releasecommon "github.com/konflux-ci/e2e-tests/tests/release"
 	releaseapi "github.com/konflux-ci/release-service/api/v1alpha1"
 	tektonutils "github.com/konflux-ci/release-service/tekton/utils"
 	tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
It seems to me that these imports were alphabetically ordered, but now they're not. Why this change?

Reply:
Format on save on the VS-Code did the auto-formatting when I saved the file. In all the files I need to disable this sometimes it formats unwanted lines.

Reply:
ok, it's fine with me. but I'd like @jinqi7 's input on this, since she's the original author of the test.